src/app/(pages)/feedback/page.jsx

```diff
@@ -1,72 +1,86 @@
 /* eslint-disable no-unused-vars */
-import React from "react";
+
+"use client";
+
+import React, { useState } from "react";
 import styles from "./feedback.module.scss";
 
 const Feedback = () => {
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const handleRadioChange = (e) => {
+    setSelectedOption(e.target.value);
```

Comment:
Do you have any values assigned? What does it return ? "Undefined".
This is where you have to use name attribute of the input